docs/concepts/copilot/copilot.md

```diff
@@ -0,0 +1,80 @@
+<div class="wistia_responsive_wrapper" style={{height:'100%',left:0,position:'absolute',top:0,width:'100%'}}>
+<iframe src="https://fast.wistia.net/embed/iframe/t7m3boc575?seo=false?videoFoam=true" title="Design a Pipeline Video" allow="autoplay; fullscreen" allowtransparency="true" frameborder="0" scrolling="no" class="wistia_embed" name="wistia_embed" msallowfullscreen width="100%" height="100%"></iframe>
+</div></div>
+<script src="https://fast.wistia.net/assets/external/E-v1.js" async></script>
+
+To learn more about what Data Copilot can do, see [Data Copilot AI capabilities](/concepts/copilot/copilot-ai-capabilities).
+
+## Knowledge Graph
+
+Data Copilot works by enhancing the user’s prompt using a knowledge graph.
+
+![Knowledge Graph](img/copilot_knowledge_graph.png)
+
+Our knowledge graph includes metadata for a Project’s entities (such as Datasets, Schemas, Seeds, Models, and Pipelines) and the statistical usages of these entities. We built the knowledge graph intentionally to include metadata but not data. The Dataset structure is included in the knowledge graph but individual Dataset records are not.
+
+![Architecture](img/copilot_arch.png)
+
+Prophecy sends the enhanced Prompt to OpenAI. The model returns SQL or Spark code, and Prophecy verifies whether this code is valid. Once Prophecy fixes the code, Prophecy generates a visual Pipeline and displays this Pipeline to you for review.
+
+## Deployment
+
+Data Copilot is available for all customers using Prophecy’s managed Public SaaS offering and uses the public SaaS version of OpenAI's Language model. Customers using the Private SaaS or on-prem offerings can enable Data Copilot as a flag in the deployment / upgrade configuration.
+
+For more details, see [Enable Data Copilot](/concepts/copilot/enable-data-copilot).
+
+## FAQ
+
+###
+
+#### How is Prophecy Copilot different than Github Copilot?
+
+Github Copilot is a great tool to boost productivity and extend the reach of the coding community. However, not every problem is solved with direct coding. More importantly, users need a Copilot with some context of the interesting data.
+
+For teams of analysts, data platform providers, or line-of-business users, the pain points are not resolved by teaching every team member how to code. Data Copilot empowers less technical users because they don’t have to code. Importantly, technical and coding users benefit from Data Copilot because visual Pipelines are easier to understand, explain, and leverage. Prophecy’s Data Copilot boosts the productivity of the business user and the more technical coding team members.
+
+When all of these data practitioners reach for an AI assistant, they’ll need one specific to the data space. For example, the assistant should collect metadata from the Databricks Workspace. Prophecy Data Copilot has the context of your data, and it can guide Pipeline and model development by suggesting which Datasets to use and how to use them.
```

Comment:
"should collect metadata from the Databricks Workspace" 
-->
"should collect metadata from the data warehouse, catalog, or storage provider"

Reply:
Done.